docs/user-guides/migrations/includes/src/main/resources/topics/ServerMigration-ServerConfiguration-Subsystem-Update-transactions.adoc

```diff
@@ -0,0 +1,32 @@
+= Update the transaction Subsystem
+
+The JBoss Server Migration Tool makes the following updates to the *transactions* subsystem configuration to better match the {server-target-productName} default configurations.
```

Comment:
The *transactions* subsystem *XML* configuration updates are necessary for such configurations to be supported by the target server